testsuite/MDAnalysisTests/coordinates/test_xdr.py

```diff
@@ -891,6 +891,11 @@ def test_unsupported_format(self, traj):
         reader[idx_frame]
 
     @pytest.mark.skipif(get_userid() == 0, reason="cannot readonly as root")
+    @pytest.mark.skipif(
+      os.name == 'nt' and "AGENT_NAME" in os.environ,
```

Comment:
Was this a problem before? If not, why is this showing up with filelock but not with fasteners?

Reply:
Is AGENT_NAME a hack for Azure runners?

Reply:
Yes the AGENT_NAME is the hack for Azure runners, so that It does not run on Azure Runners. As to why it
does not run on Azure runners, I was not able to track any reasoning behind it, the only thing that I found that
was remotely similar was #4122.

Reply:
Please add a comment to explain the hack.